dev-db/cpp-driver/cpp-driver-2.4.3.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-utils
+
+DESCRIPTION="DataStax C/C++ Driver for Cassandra"
+HOMEPAGE="http://datastax.github.io/cpp-driver/"
+SRC_URI="https://github.com/datastax/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+RESTRICT="nomirror"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-libs/libuv
+	|| (
+		dev-libs/openssl:0
+		dev-libs/libressl
+	)
```

Comment:
this is incorrect, please read how this should be done on the project page: https://wiki.gentoo.org/wiki/Project:LibreSSL

You'll need to add `libressl` to `IUSE` and then in `RDEPEND` do

```
!libressl? ( dev-libs/openssl:0= )
libressl? ( dev-libs/libressl:0= )
```